Python/pywarpx/picmi.py

```diff
@@ -1341,12 +1345,15 @@ class EmbeddedBoundary(picmistandard.base._ClassWithInit):
         Analytic expression defining the potential. Can only be specified
         when the solver is electrostatic.
 
+    cover_multiple_cuts: bool, default=False
+        Whether to cover cells with multiple cuts.
+        (If False, this will raise an error if some cells have multiple cuts)
 
     Parameters used in the analytic expressions should be given as additional keyword arguments.
 
     """
     def __init__(self, implicit_function=None, stl_file=None, stl_scale=None, stl_center=None, stl_reverse_normal=False,
-                 potential=None, **kw):
+                 potential=None, cover_multiple_cuts=False, **kw):
```

Comment:
The default value of `cover_multiple_cuts` should be None. This avoids having the default value set in two different places, here and in the C++.